packages/example/src/pages/guides/configuration.mdx

```diff
@@ -148,14 +148,16 @@ To add a link to the bottom of each page that points to the current page source
 
 - `additionalFontWeights` – add support for additional Plex font weights. Don't forget to specify italics for the additional weights if needed.
 - `mdxExtensions` – change the file extensions processed by `gatsby-mdx` (default ['.mdx', '.md'])
+- `titleType` – the format of the title in your head. `site` for your site title, `page` for your individual page title, or `append` for "title – pageTitle" (default 'page')
```

Comment:
I think we need some examples here to make it easier to understand what each one means.

Reply:
@alisonjoseph  added